deploy/rp-production.json

```diff
@@ -236,6 +236,49 @@
                 "[resourceId('Microsoft.Network/loadBalancers', 'rp-lb')]"
             ]
         },
+        {
+            "properties": {
+                "severity": 4,
+                "enabled": true,
+                "scopes": [
+                    "[resourceId('Microsoft.Network/loadBalancers', 'rp-lb')]"
+                ],
+                "evaluationFrequency": "PT5M",
+                "windowSize": "PT1H",
+                "targetResourceType": "Microsoft.Network/loadBalancers",
+                "criteria": {
+                    "allOf": [
+                        {
+                            "operator": "LessThan",
+                            "threshold": 99.95,
```

Comment:
@ehashman @Makdaam does this mean "fire if the RP was unreachable for > 1.8 seconds in the previous hour"?  Is that a bit extreme?

Reply:
Changed to 95% in 1h... so 3min.